plenum/common/strategies.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from statistics import median_low, median, median_high
+from typing import List
+
+from plenum.common.moving_average import EMAEventFrequencyEstimator
+
+
+class AverageStrategyBase(ABC):
```

Comment:
I would separate average strategies and monitor strategies into separate files

Reply:
ok